spring/src/test/java/org/axonframework/spring/config/EventProcessingConfigurationConfigTest.java

```diff
@@ -52,19 +53,20 @@ public void testEventProcessingConfiguration() {
         assertTrue(eventProcessingConfiguration.eventProcessor("processor2").isPresent());
         assertTrue(eventProcessingConfiguration.eventProcessor("subscribingProcessor").isPresent());
 
-        assertEquals("processor2",
-                     eventProcessingConfiguration.eventProcessorByProcessingGroup("processor1").get().getName());
+        EventProcessor processor2 = eventProcessingConfiguration.eventProcessorByProcessingGroup("processor1").get();
+        assertEquals("processor2", processor2.getName());
+        List<MessageHandlerInterceptor<? super EventMessage<?>>> interceptorsFor = eventProcessingConfiguration
+                .interceptorsFor("processor2");
+        assertEquals(2, interceptorsFor.size());
+        assertTrue(interceptorsFor.stream().anyMatch(i -> i instanceof CorrelationDataInterceptor));
+        assertTrue(interceptorsFor.stream().anyMatch(i -> i instanceof LoggingInterceptor));
+
         assertEquals("processor2",
                      eventProcessingConfiguration.eventProcessorByProcessingGroup("processor2").get().getName());
         assertEquals("subscribingProcessor",
                      eventProcessingConfiguration.eventProcessorByProcessingGroup("processor3").get().getName());
         assertEquals("subscribingProcessor",
                      eventProcessingConfiguration.eventProcessorByProcessingGroup("Saga3Processor").get().getName());
-        List<MessageHandlerInterceptor<? super EventMessage<?>>> interceptors = eventProcessingConfiguration
```

Comment:
Why is this assertion for `processor3` removed?

Reply:
I now do the check for processor2 instead of processor3. I could do the interceptor check for both, but just picked one.